### DISCUSSION TITLE
Editable forms in Sunbird

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#732] [General Discussion]
Hello Team,

We are trying to consolidate forms in Sunbird which can be customised. Here are a few lists of them, and are there any more forms that can be customized? 

1. Change the label of BMGS of home tab and profile page.
2. Course creation form -> we can add new fields, remove fields and change the labels.
3. Collection creation form -> we can add new fields, remove fields and change the labels.
4. Question Set creation form -> we can add new fields, remove fields and change the labels.
5. Send for review form.
6. Add new tab buttons.
 
 cc:  @AmiableAnil @Hari-stackroute @ashokreddy1208 @vinukumar-vs @rajeevsathish @rajeshkumaravel @akulavasu @amitpriyadarshi

**Comment by HarishGangula** [comment]
@siddhishinde0723  You can refer this https://ed.sunbird.org/v/release-6.0.0/use/source-code/apis#sunbird-ed-portal-postman-forms-config-documentation

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
